tests/image_test.py

```diff
@@ -1225,5 +1226,29 @@ def test_load_img(self, tmpdir):
                                        interpolation="unsupported")
 
 
+    def test_list_pictures(self, tmpdir):
+        images = [os.path.join(tmpdir, 'test.png')]
```

Comment:
you can create all the images in a single list comprehension as I indicated in my other comment